conda/lock.py

```diff
@@ -36,22 +54,16 @@ class Locked(object):
     def __init__(self, path, retries=10):
         self.path = path
         self.end = "-" + str(os.getpid())
-        self.lock_path = os.path.join(self.path, LOCKFN + self.end)
+        self.lock_path = join(self.path, LOCKFN + self.end)
         self.retries = retries
 
     def __enter__(self):
-        # Keep the string "LOCKERROR" in this string so that external
-        # programs can look for it.
-        lockstr = ("""\
-    LOCKERROR: It looks like conda is already doing something.
-    The lock %s was found. Wait for it to finish before continuing.
-    If you are sure that conda is not running, remove it and try again.
-    You can also use: $ conda clean --lock\n""")
+
         sleeptime = 1
 
         for _ in range(self.retries):
```

Comment:
`range` is now being imported from `conda.compat`, because in python2 we should be using `xrange`.